src/coordsim/reader/reader.py

```diff
@@ -223,4 +223,5 @@ def read_network(file, node_cap=None, link_cap=None):
 
 def reset_cap(network):
     for node in network.nodes.items():
-        node[1]['remaining_cap'] = node[1]['cap']
+        network.nodes[node[0]]['remaining_cap'] = network.nodes[node[0]]['cap']
```

Comment:
Why not just iterate over `network.nodes.keys()` rather than `.items()` and then indexing `[0]` to get the key?